src/api/java/com/minecolonies/api/colony/fields/registry/FieldRegistries.java

```diff
@@ -0,0 +1,139 @@
+package com.minecolonies.api.colony.fields.registry;
+
+import com.minecolonies.api.IMinecoloniesAPI;
+import com.minecolonies.api.colony.IColony;
+import com.minecolonies.api.colony.IColonyView;
+import com.minecolonies.api.colony.fields.IField;
+import com.minecolonies.api.colony.fields.IFieldMatcher;
+import com.minecolonies.api.colony.fields.IFieldView;
+import com.minecolonies.api.util.constant.Constants;
+import net.minecraft.core.BlockPos;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraftforge.registries.IForgeRegistry;
+import net.minecraftforge.registries.RegistryObject;
+
+import java.util.function.BiFunction;
+
+/**
+ * Registry implementation for field instances.
+ */
+public class FieldRegistries
+{
+    public static final ResourceLocation FARM_FIELD_ID       = new ResourceLocation(Constants.MOD_ID, "farmfield");
+    public static final ResourceLocation PLANTATION_FIELD_ID = new ResourceLocation(Constants.MOD_ID, "plantationfield");
+
+    public static RegistryObject<FieldRegistries.FieldEntry> farmField;
+    public static RegistryObject<FieldRegistries.FieldEntry> plantationField;
+
+    private FieldRegistries()
+    {
+    }
+
+    /**
+     * Get the field registry.
+     *
+     * @return the reward registry.
+     */
+    public static IForgeRegistry<FieldEntry> getFieldRegistry()
+    {
+        return IMinecoloniesAPI.getInstance().getFieldRegistry();
+    }
+
+    /**
+     * Entry for the {@link IField} registry. Makes it possible to create a single registry for a {@link IField}. Used to lookup how to create {@link IField} and {@link
+     * IFieldView}.
+     */
+    public static class FieldEntry
```

Comment:
I would get rid of the field type enum and just make the field type enum a Resourcelocation, as we already got this here. Its one FieldEntry per enum type, right? So a resource location per enum is enough.